pkg/controllers/networkinfo/vpcnetworkconfig_handler.go

```diff
@@ -25,11 +25,12 @@ import (
 // - VPC Network Configuration update:	Only support updating external/private ipblocks, update values in cache
 
 type VPCNetworkConfigurationHandler struct {
-	Client     client.Client
-	vpcService commontypes.VPCServiceProvider
+	Client              client.Client
+	vpcService          commontypes.VPCServiceProvider
+	ipBlocksInfoService commontypes.IPBlocksInfoServiceProvider
 }
 
-func (h *VPCNetworkConfigurationHandler) Create(_ context.Context, e event.CreateEvent, _ workqueue.RateLimitingInterface) {
+func (h *VPCNetworkConfigurationHandler) Create(ctx context.Context, e event.CreateEvent, _ workqueue.RateLimitingInterface) {
 	vpcConfigCR := e.Object.(*v1alpha1.VPCNetworkConfiguration)
 	vname := vpcConfigCR.GetName()
 	ninfo, _err := buildNetworkConfigInfo(*vpcConfigCR)
```

Comment:
We can change `_err` here, and use `err`.

Reply:
Updated, thanks!